app/assets/scripts/map.js

```diff
@@ -19,9 +19,13 @@ function markerListener(marker, event) {
   var markerInfoWindowTemplate = $("#marker-info-window-template").html();
   var compiledMarkerInfoWindowTemplate =
     Handlebars.compile(markerInfoWindowTemplate);
+  if(marker.avg_rating == -1){
```

Comment:
chose to set to 0 because handlebars only checks for existance, not for matching a value!